components/input.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export function Input ({
```

Comment:
That a bit too generic :) 
This will produce and `input type="text"` field
So to avoid confusion and be precise, should this be:
```suggestion
export function TextInput ({
```

Reply:
This one can be changed via props, right? I feel like `text` is a sane default for it.